examples/api/filled_step.py

```diff
@@ -172,7 +172,7 @@ def stack_hist(ax, stacked_data, sty_cycle, bottoms=None,
 hist_func = partial(np.histogram, bins=edges)
 
 # set up style cycles
-color_cycle = cycler('facecolor', 'rgbm')
+color_cycle = cycler('facecolor', plt.rcParams['axes.prop_cycle'][:4])
```

Comment:
Yes, good point; I suppose I'll have to iterate, unless the `by_key` thing you mentioned in the other comment is released.